### PR TITLE
chore(arrow): bump apache-arrow to 7.0.0

### PR DESCRIPTION
**Pull request by ibgreen**
Arrow 5 upgrade is not working so checking out #1753.

### CODE REVIEW

**Comment by trxcllnt** [comment]
@ibgreen I believe this build error should be solved in `apache-arrow@6.0.1`

File: modules/arrow/fixed-package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "flatbuffers": "1.12.0",
+    "json-bignum": "^0.0.3",
+    "pad-left": "^2.1.0",
+    "text-encoding-utf-8": "^1.0.2",
+    "tslib": "^2.2.0"
+  },
+  "bin": {
+    "arrow2csv": "bin/arrow2csv.js"
+  },
+  "browser": {
```
Comment (ibgreen) [inline modules/arrow/fixed-package.json:45]: @trxcllnt If I add this my bundling works, but I assume it overwrites your existing browser field.

Comment (trxcllnt) [reply modules/arrow/fixed-package.json:45]: I believe we're publishing v6.0.2 to fix this packaging issue.

Comment (trxcllnt) [reply modules/arrow/fixed-package.json:45]: https://github.com/apache/arrow/pull/11742

**Comment by ibgreen** [comment]
@trxcllnt Looks mostly good. Noticed a crash in the new fancy RecordBatch constructor. Probably need to massage the data a bit more?

**Comment by trxcllnt** [comment]
Updating to v8.0.0 might fix some of the issues we were seeing here

**Comment by kylebarron** [comment]
Can probably be closed after https://github.com/visgl/loaders.gl/pull/2276

**Comment by ibgreen** [comment]
> Can probably be closed after #2276

Yes. Though the arrow update is a breaking change and currently landed on 4.0-dev without a clear timeline for completion.

**Pull request closed without merging**